pkgs/development/python-modules/wifi/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pbkdf2
+, pythonOlder
+, substituteAll
+, wirelesstools
+}:
+
+buildPythonPackage rec {
+  pname = "wifi";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "rockymeza";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-scg/DvApvyQZtzDgkHFJzf9gCRfJgBvZ64CG/c2Cx8E=";
+  };
+
+  disabled = pythonOlder "2.6";
+
+  patches = [
+    (substituteAll {
+      src = ./iwlist-path.patch;
+      inherit wirelesstools;
+    })
+  ];
+
+  propagatedBuildInputs = [
+    pbkdf2
+  ];
+
```

Comment:
Please run tests.

Reply:
They were running without defining `doCheck = true`. I've added it.

Reply:
I didn't mean that but I think you couldn't know it. The tests are run trough `python setup.py test` which is depreacted by upstream. Ideally we want to run the tests with pytestCheckHook if possible.

Reply:
Thx for the info